RevenueCatUI/CustomerCenter/Views/RelevantPurchasesListView.swift

```diff
@@ -234,44 +213,37 @@ struct RelevantPurchasesListView: View {
 
     @ViewBuilder
     private var accountDetailsView: some View {
-        Text(localization[.accountDetails].uppercased())
-            .font(.subheadline)
-            .foregroundStyle(.secondary)
-            .padding(.horizontal, 32)
-            .frame(maxWidth: .infinity, alignment: .leading)
-            .multilineTextAlignment(.leading)
-            .padding(.top, 32)
-            .padding(.bottom, 16)
-
-        VStack {
-            if let originalPurchaseDate = viewModel.originalPurchaseDate {
-                CompatibilityLabeledContent(
-                    localization[.dateWhenAppWasPurchased],
-                    content: dateFormatter.string(from: originalPurchaseDate)
-                )
+        ScrollViewSection(title: localization[.accountDetails]) {
+            VStack {
+                if let originalPurchaseDate = viewModel.originalPurchaseDate {
+                    CompatibilityLabeledContent(
+                        localization[.dateWhenAppWasPurchased],
+                        content: dateFormatter.string(from: originalPurchaseDate)
+                    )
 
-                Divider()
-            }
+                    Divider()
+                }
 
-            CompatibilityLabeledContent(
-                localization[.userId],
-                content: viewModel.originalAppUserId
-            )
-            .contextMenu {
-                Button {
-                    UIPasteboard.general.string = viewModel.originalAppUserId
-                } label: {
-                    Text(localization[.copy])
-                    Image(systemName: "doc.on.clipboard")
+                CompatibilityLabeledContent(
+                    localization[.userId],
+                    content: viewModel.originalAppUserId
+                )
+                .contextMenu {
+                    Button {
+                        UIPasteboard.general.string = viewModel.originalAppUserId
+                    } label: {
+                        Text(localization[.copy])
+                        Image(systemName: "doc.on.clipboard")
+                    }
                 }
             }
+            .padding()
+            .background(Color(colorScheme == .light
+                              ? UIColor.systemBackground
+                              : UIColor.secondarySystemBackground))
+            .cornerRadius(10)
+            .padding(.horizontal)
         }
-        .padding()
-        .background(Color(colorScheme == .light
-                          ? UIColor.systemBackground
-                          : UIColor.secondarySystemBackground))
-        .cornerRadius(10)
-        .padding(.horizontal)
     }
 
     private var buttonsView: some View {
```

Comment:
Not something done in this PR, but I just realized that `buttonsView` is not being used